python/src/aiconfig/editor/client/src/components/prompt/prompt_input/schema_renderer/PromptInputSchemaRenderer.tsx

```diff
@@ -77,6 +80,7 @@ export default memo(function PromptInputSchemaRenderer(props: Props) {
         label="Prompt"
         onChange={(e) => props.onChangeInput(e.target.value)}
         placeholder="Type a prompt"
+        disabled = {readOnly}
```

Comment:
same here

Reply:
And here :p

Reply:
Same about making placeholder empty string for readOnly mode

Reply:
readOnly = False
<img width="1085" alt="Screenshot 2024-01-19 at 10 28 40 AM" src="https://github.com/lastmile-ai/aiconfig/assets/141073967/2bb1babd-6799-433c-8873-0d5bd3ffc8d7">

readOnly = True
<img width="990" alt="Screenshot 2024-01-19 at 10 28 54 AM" src="https://github.com/lastmile-ai/aiconfig/assets/141073967/2ea4c77d-d79b-4ece-a864-98b28baed89e">

placeHolder no longer needed since spoiler will render input which will be an undefined/empty string